std/mmfile.d

```diff
@@ -612,23 +624,45 @@ private:
     {
         static assert(0);
     }
+}
+
+/// Read an existing file
+@system unittest
+{
+    import std.file;
+    std.file.write(deleteme, "hello"); // deleteme is a temporary filename
+    scope(exit) remove(deleteme);
+
+    // Use a scope class so the file will be closed at the end of this function
+    scope mmfile = new MmFile(deleteme);
+
+    assert(mmfile.length == "hello".length);
+
+    // Access file contents with the slice operator
+    // This is typed as `void[]`, so cast to `char[]` or `ubyte[]` to use it
+    const data = cast(const(char)[]) mmfile[];
+
+    // At this point, the file content hasn't been read yet.
+    // The OS will
```

Comment:
Is this an incomplete sentence?

Reply:
Good catch, https://github.com/dlang/phobos/pull/8892